src/reference/asciidoc/metrics.adoc

```diff
@@ -165,6 +165,20 @@ It is possible to customize the names and tags of `Meters` created by integratio
 The https://github.com/spring-projects/spring-integration/blob/master/spring-integration-core/src/test/java/org/springframework/integration/support/management/micrometer/MicrometerCustomMetricsTests.java[MicrometerCustomMetricsTests] test case shows a simple example of how to do that.
 You can also further customize the meters by overloading the `build()` methods on builder subclasses.
 
+Starting with version 5.1.13, the `QueueChannel` exposes Micrometer gauges for queue size and remaining capacity:
```

Comment:
Well, it even misleads you when you try to upgrade from `5.1.13` to those `5.2.8` or `5.3.2`, because there is no that fix yet.
So, it was always about upgrading to the latest minor version of the particular point generation.